matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/crypto/DefaultCryptoService.kt

```diff
@@ -856,8 +580,16 @@ internal class DefaultCryptoService @Inject constructor(
             return
         }
         cryptoCoroutineScope.launch(coroutineDispatchers.crypto) {
-            val userIds = getRoomUserIds(roomId)
-            setEncryptionInRoom(roomId, event.content?.get("algorithm")?.toString(), true, userIds)
+            val params = LoadRoomMembersTask.Params(roomId)
+            try {
```

Comment:
This would defeat lazy loading of room members no?

Reply:
This was already the case before I got here: https://github.com/vector-im/element-android/commit/af338b06078d790d7cf22a56deda510f285f3645#diff-10f33a2aedee42d5ed7a9b1db212f3f0992a6852592e700d04e66ca34ad2beaaR764